__tests__/src/middleware.spec.js

```diff
@@ -70,7 +71,7 @@ describe('pipe', () => {
   });
 
   it('should show a message when there is a socket error', () => {
-    jest.spyOn(console, 'error');
+    console.error = jest.fn();
```

Comment:
why this change?

Reply:
There was an issue where jest would pick up the console.error and report it back as a test failure? even though we spy on console.error. so Matt suggested trying that out and it worked fine including snapshots for it.